dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/Ds3JobTask.java

```diff
@@ -116,38 +113,12 @@ void getTransferRates(final Instant jobStartInstant, final AtomicLong totalSent,
         updateProgress(totalSent.get() / 2, totalJobSize);
     }
 
-    void updateInterruptedJobsBtn(final JobInterruptionStore jobInterruptionStore, final UUID jobId) {
-        final Map<String, FilesAndFolderMap> jobIDMap = ParseJobInterruptionMap.getJobIDMap(
-                jobInterruptionStore.getJobIdsModel().getEndpoints().stream().collect(GuavaCollectors.immutableList()),
-                ds3Client.getConnectionDetails().getEndpoint(),
-                deepStorageBrowserPresenter.getJobProgressView(), jobId);
-        if (currentSession != null) {
-            final String currentSelectedEndpoint = currentSession.getEndpoint() + COLON + currentSession.getPortNo();
-            if (currentSelectedEndpoint.equals(ds3Client.getConnectionDetails().getEndpoint())) {
-                ParseJobInterruptionMap.setButtonAndCountNumber(jobIDMap, deepStorageBrowserPresenter);
-            }
-        }
-    }
-
-    void removeJobIdAndUpdateJobsBtn(final JobInterruptionStore jobInterruptionStore, final UUID jobId) {
-        final Map<String, FilesAndFolderMap> jobIDMap = ParseJobInterruptionMap.removeJobID(jobInterruptionStore, jobId.toString(), ds3Client.getConnectionDetails().getEndpoint(), deepStorageBrowserPresenter, loggingService);
```

Comment:
This is side effecting when called, so I'm not confident that we can remove this here.

Reply:
The side effect was moved into the RecoverInteruptedJob code since it belongs there.